topics/admin/tutorials/pulsar/tutorial.md

```diff
@@ -56,8 +56,9 @@ At the Galaxy end, it is configured within the `job_conf.xml` file and uses one
 
 In this tutorial, we will:
 
+* Install and configure a message queueing system on our Galaxy server (can be a different VM)
 * Install and configure a Pulsar server on a remote linux machine using ansible
-    * We will configure the Pulsar server to run via the RESTful interface
+    * We will configure the Pulsar server to run via the message queueing interface
 * Configure our Galaxy servers to run a job there
 * Run a job remotely
```

Comment:
This is basically the agenda, let's just remove this and fix the titles to match?

Reply:
Sounds good